### ISSUE TITLE
微调时，数据已经成功读取，finetune_moss.py报错TypeError: pad_sequence(): argument 'sequences' (position 1) must be tuple of Tensors, not Tensor

### ISSUE THREAD

**Issue by lizhidomg** [OpenLMLab/MOSS#265]
SFT微调时，数据已经成功读取，但是finetune_moss.py运行到for batch_cnt, (input_ids, attention_mask, labels) in enumerate(train_dataloader):行时报错，
![QQ截图20230511124940](https://github.com/OpenLMLab/MOSS/assets/55686901/46d1b9d7-0c2a-4de2-a1df-0a086ae5829b)
数据的格式是这样的，总共100个
![QQ截图20230511125400](https://github.com/OpenLMLab/MOSS/assets/55686901/32ac54da-8ac7-44c5-82f3-687363235652)
